ats/pkb/parents.py

```diff
@@ -0,0 +1,14 @@
+def parent(list):
```

Comment:
To samo, jak query preprocessor ma sie do tego odwolac przy analogicznym zapytaniu jak w `follows.py`